src/main/java/com/google/sps/ImageTextDectector.java

```diff
@@ -16,45 +16,109 @@
 
 import com.google.sps.data.ShoppingListTextEntry;
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 
 /**
- * This class generates a shopping list based on image 
- * 1) It uses cloudVisionAPI to scan an image containing shopping
+ * This class generates a shopping list based on image provided in method 'extractShoppingList'
```

Comment:
I found this [link](https://stackoverflow.com/questions/5915992/how-to-reference-a-method-in-javadoc) useful, so I'm suggesting this change:
```suggestion
 * This class generates a shopping list based on image provided in {@link #extractShoppingList(byte[])}
```